README.md

```diff
@@ -1 +1,6 @@
-# Huxflux
+
+This is my solution to the February 2021 code challenge: HuxFlux Function. The challenge is arranged by tjejerkodar.se in collaboration with svenskaspel.se.
+
+For this challenge I didn´t changed anything in the user interface, I did the logic only, so my code contribution is in huxflux.js.
```

Comment:
Good job! I really hope that you continue with the challenges and maybe next time try to make som changes to the styling (user interface) 😄 👍🏻

Reply:
Thank you for the feedback! :)